charts/retool/templates/_helpers.tpl

```diff
@@ -351,3 +351,10 @@ Usage: (template "retool.codeExecutor.image.tag" .)
 {{- end -}}
 {{- $output -}}
 {{- end -}}
+
+{{/*
+Checks whether or not ExternalSecret definitions are enabled and can potentially clobber secrets or explicitly allow additional direct secret refs.
+*/}}
+{{- define "checkExternalSecretsClobbering" -}}
```

Comment:
I would name this something a little more directly tied to what it gates, like `shouldIncludeConfigSecretsEnvVars` or smth

Reply:
agree